src/cow-react/modules/trade/types/TradeFullState.ts

```diff
@@ -1,7 +1,7 @@
 import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
 import { OrderKind } from '@cowprotocol/cow-sdk'
 
-export interface TradeWidgetState {
+export interface TradeFullState {
```

Comment:
I find the suffix `fullState` confusing.
What's the difference between raw and full states?

Reply:
Renamed, thank you!